app/cli.ts

```diff
@@ -1,16 +1,45 @@
+import DB, { insertTestRun } from "./db.js";
+
 /**
  * The help message that is written once --help is given as an argument
  */
 const help = `
 --help         :: Prints this message
 --host=<value> :: Sets the host of the server (default = 127.0.0.1)
 --port=<value> :: Sets the port of the server (default = 3000)
+--mariadb-user=<value>     :: Sets the user of the mariadb connection (default = energylabel)
+--mariadb-password=<value> :: Sets the password of the mariadb connection (default = energylabel)
+--mariadb-database=<value> :: Sets the database of the mariadb connection (default = energylabel)
+--mariadb-port=<value> :: Sets the port of the mariadb connection (default = 3306)
+--mariadb-host=<value> :: Sets the port of the mariadb connection (default = localhost)
+--mariadb-init :: Initializes the database
+--mariadb-column-store=<bool> :: Sets the storage to column store in --mariadb-init (default = true)
 `;
 
 /**
  * The valid argument keys of the server
  */
-const validArgs = ["--host", "--port"] as const;
+const validArgs = [
+	"--host",
+	"--port",
+	"--mariadb-user",
+	"--mariadb-password",
+	"--mariadb-database",
+	"--mariadb-port",
+	"--mariadb-host",
+	"--mariadb-column-store",
+] as const;
+
+/**
+ * The valid single arguments,
+ * NOTE CREATE A CODITION AT THE END OF THIS FILE
```

Comment:
Er dette blevet gjort?

Reply:
For at sige a hvis man ændre dette array så skal man også tilføje det til slutningen af filen.